org/postgresql/core/Oid.java

```diff
@@ -83,7 +83,8 @@
     public static String toString(int oid) {
         try {
             Field[] fields = Oid.class.getFields();
-            for (Field field : fields) {
```

Comment:
Ill hange it back to foreach loop. Although there is minor perf penalty over vanilla for loop